src/types/domain.d.ts

```diff
@@ -355,6 +355,11 @@ export interface Event {
   relatedService?: Partial<InformationModel>[];
 }
 
+export interface EventDTO extends Event {
```

Comment:
Hvorfor lage EventDTO? Vi har jo Event allerede

Reply:
Det er for å ta i mot events riktig der type er SkosConcept[] og eventType  er string, og så bytte de om så events følger mønsteret til resten av entitetene - altså at `type` er en string som bestemmer entitetstype.

Alternativet er å legge til special case for events om det er mer ønskelig?